src/docfx/build/manifest/FileManifest.cs

```diff
@@ -10,5 +13,13 @@ internal class FileManifest
         public string OutputPath { get; set; }
 
         public string SourcePath { get; set; }
+
+        public List<string> Monikers { get; set; }
+
+        [JsonIgnore]
+        public string MonikerSeg { get; set; }
+
+        [JsonIgnore]
+        public Document File { get; set; }
```

Comment:
can we remove `Document File` from this class, I want our data contract to be plain old c# class without [JsonIgnore], there are other ways to pass objects around without injecting objects to data contract.